dev/tests/integration/testsuite/Magento/GraphQl/Quote/_files/enable_offline_shipping_methods.php

```diff
@@ -3,6 +3,12 @@
  * Copyright © Magento, Inc. All rights reserved.
```

Comment:
Found the use of this file in `\Magento\GraphQl\AuthorizenetAcceptjs\Customer\SetPaymentMethodTest::testPlaceOrder`. It needs to be replaced with `@magentoConfigFixture`.

Reply:
Done